opentracing_instrumentation/client_hooks/requests.py

```diff
@@ -93,4 +104,9 @@ def host_port(self):
             return split_host_and_port(host_string=self.host_str,
                                        scheme=self.scheme)
 
-    requests.adapters.HTTPAdapter.send = send_wrapper
+
+requests_patcher = RequestsPatcher()
+
+# install_patches is deprecated,
+# kept for backward compatibility
+install_patches = requests_patcher.install_patches
```

Comment:
I think this needs to be a function so that `requests_patcher` is resolved late, not during module load (in case users want to overwrite it).